android/expoview/src/main/java/host/exp/exponent/experience/ExperienceActivity.java

```diff
@@ -317,6 +317,18 @@ public void onNewIntent(Intent intent) {
     }
   }
 
+  /**
+   * Handles command line command `adb shell input keyevent 82` that toggles the dev menu on the current experience activity.
+   */
+  public boolean toggleDevMenu() {
+    if (mReactInstanceManager != null && mReactInstanceManager.isNotNull() && !mIsCrashed) {
+      mDevMenuManager.toggleInActivity(this);
+      return true;
+    }
+    return false;
+  }
```

Comment:
Is this method called when the user runs `adb shell input keyevent 82`? If no, change the comment ;)